src/main/resources/application.properties

```diff
@@ -55,7 +55,6 @@ onelogin.service-redirect-url=redirectUrlValue
 onelogin.private-key=privateKeyValue
 feature.onelogin.enabled=false
 feature.onelogin.migration.enabled=false
-feature.validate-user-roles-in-middleware=true
```

Comment:
Why are we removing this?

Reply:
Another develop merge oddity, I'm looking into it

Reply:
Reverted :)